src/pages/SignupPage/SignUpPage.jsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import { Navbar, Footer, Input, Label } from "../../components";
+import { Link } from "react-router-dom";
+import "./sign-up-page.css";
+import { BiEye, BiEyeSlash } from "../../assets/icons";
+
+export function SignUpPage() {
+  return (
+    <div classNameName='signup-container'>
+      <Navbar />
+      <main className='signup-main flex-column'>
+        <div className='form-container card'>
+          <h2 className='form-heading'>Sign Up</h2>
+          <form>
+            <div className='form-set'>
+              <Label labelFor='fname' labelName='First Name' />
+              <Input
+                type={"text"}
```

Comment:
Suggestion - just quotes would also work. You can remove the curly braces as it's just a string.

Reply:
Thanks @foolhardy21 for pointing that will update it.